client/lib/analytics/README.md

```diff
@@ -14,7 +14,7 @@ You can limit to only calls made to GA, Tracks, or MC by replacing the `*` with
 
 ## Which analytics tool should I use?
 
-*Page Views* (`analytics.pageView.record`) should be used to record all page views (when the main content body completely changes). This will automatically record the pageview to both Google Analytics and Tracks.
+*Page Views* should be used to record all page views (when the main content body completely changes). This will automatically record the pageview to both Google Analytics and Tracks. The preferred tool to record page views is the [`PageViewTracker`](https://github.com/Automattic/wp-calypso/tree/master/client/lib/analytics/page-view-tracker) component.
```

Comment:
I think we can expand on _why_ this is preferred for example going into detail about site timing, and verifying if a site fragment is a site.

We should also explicitly warn against page view calls calls in the controller. (We may want to consider turning this into an eslint rule).

Reply:
> We should also explicitly warn against page view calls calls in the controller. (We may want to consider turning this into an eslint rule).

+1